src/Datadog.Trace/Agent/StreamFactories/NamedPipeClientStreamFactory.cs

```diff
@@ -0,0 +1,38 @@
+using System.IO;
+using System.IO.Pipes;
+
+namespace Datadog.Trace.Agent.StreamFactories
+{
+    internal class NamedPipeClientStreamFactory : IStreamFactory
+    {
+        private readonly string _pipeName;
+        private readonly string _serverName;
+        private readonly PipeOptions _pipeOptions;
+        private readonly int _timeoutMs;
+
+        public NamedPipeClientStreamFactory(string pipeName, int timeoutMs)
+            : this(pipeName, ".", PipeOptions.Asynchronous, timeoutMs)
+        {
+        }
+
+        public NamedPipeClientStreamFactory(string pipeName, string serverName, PipeOptions pipeOptions, int timeoutMs)
+        {
+            _pipeName = pipeName;
+            _serverName = serverName;
+            _pipeOptions = pipeOptions;
+            _timeoutMs = timeoutMs;
+        }
+
+        public string Info()
+        {
+            return $@"\\\\{_serverName}\\pipe\\{_pipeName}";
```

Comment:
Since you added the `@`, I believe it should be `"\\{_serverName}\pipe\{_pipeName}"`